moveit_core/robot_trajectory/test/test_robot_trajectory.cpp

```diff
@@ -150,6 +151,210 @@ class RobotTrajectoryTestFixture : public testing::Test
   }
 };
 
+class OneRobot : public testing::Test
+{
+protected:
+  void SetUp() override
+  {
+    static const std::string MODEL2 =
```

Comment:
Note - I pretty much copied this model from here https://github.com/ros-planning/moveit2/blob/main/moveit_core/robot_state/test/test_cartesian_interpolator.cpp#L57